velox/exec/tests/TableScanTest.cpp

```diff
@@ -190,7 +191,16 @@ TEST_P(TableScanTest, allColumns) {
   writeToFile(filePath->path, kTableScanTest, vectors);
   createDuckDbTable(vectors);
 
-  assertQuery(tableScanNode(), {filePath}, "SELECT * FROM tmp");
+  auto plan = tableScanNode();
+  auto task = assertQuery(plan, {filePath}, "SELECT * FROM tmp");
+
+  // A quick sanity check for memory usage reporting. Check that peak total
+  // memory usage for the project node is > 0.
```

Comment:
Can't we run a single driver and use the actual `peakMemoryBytes` to verify?

Reply:
@majetideepak I'm not sure I understand the suggestion. When debugging memory efficiency, it is important to understand how much memory is used by each operator. Knowing the total memory across operator is useful, but not sufficient for effective troubleshooting.